.github/workflows/build-and-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and lint
+
+on:
+  pull_request:
+
+jobs:
+  sdk:
+    name: lint sdk
+    runs-on: ubuntu-20.04
+    env:
+      TEST_PATH: /tmp/test-results
+    steps:
+      - name: Checkout sdk
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint sdk
+        run: |
+          yarn build --reporter-options output=$TEST_PATH/sdk.xml
+          yarn lint --format junit -o $TEST_PATH/sdk-lint.xml
```

Comment:
Do you know how we can access these artifacts in the github UI?

Reply:
I've added a package to upload the artifacts. You can download at the bottom of the summary page:
https://github.com/OffchainLabs/arbitrum-sdk/actions/runs/3014985047

Reply:
Also added a test report page:
https://github.com/OffchainLabs/arbitrum-sdk/runs/8248976003

Reply:
nice!